src/docs/index.md

```diff
@@ -128,6 +128,96 @@ title: Akka.NET Documentation
 - [Scheduler](Scheduler)
 - [Circuit Breaker](CircuitBreaker)
 
+### Akka Streams (work in progress)
+*we are currently in the progress of porting the streams documentation for Akka.Net. As such not all sections will be available*
+
+- [Introduction](streams/introduction)
+	- [Motivation](streams/introduction#motivation)
+	- [How to read these docs](streams/introduction#how-to-read-these-docs)
+- [Quick Start Guide](streams/quickstart)
+	- [Reusable Piece](streams/quickstart#reusable-pieces)
+	- [Time-Based Processing](streams/quickstart#time-based-processing)
+- [Reactive Tweets](streams/reactivetweets)
```

Comment:
I don't know if you'll find easy and feasible implementation of .NET twitter API for this example.